### ISSUE TITLE
Server: "host" and "port" command line arguments are ignored

### ISSUE THREAD

**Issue by cedx** [cedx/akismet.js#4]
When I try to start a server instance on a different host or port, the created server is still using the defaults:

```shell
$ akismet --host=127.0.0.1 --port=9000
[Mon, 28 Dec 2015 01:55:37 GMT] Akismet server listening on 0.0.0.0:3000
```
It seems that this parameter is completely ignored by the command line parser.

**Comment by cedx** [comment]
This also affects the `port` parameter :(

**Comment by cedx** [comment]
Fixed in version 0.7.1.
